app/models/planning_application.rb

```diff
@@ -216,6 +216,20 @@ def parsed_proposal_details
     proposal_details.present? ? JSON.parse(proposal_details) : []
   end
 
+  def proposals_with_metadata
```

Comment:
I think 'proposals' is the wrong word here really. I think `proposal_details` is probably wrong too, but I think calling each question/answer a proposal is a further step into confusing words. Struggling to think of something else though. `proposal_questions_with_metadata`? `proposal_detail_answers_with_metadata`? What do you think?

Reply:
Or I guess just `proposal_details_with_metadata`. That's at least consistent.